src/cldfbench/dataset_template/.github/workflows/python-package.yml

```diff
@@ -0,0 +1,29 @@
+name: CLDF-validation
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This should be `main` rather than `master`, because that's the new default branch name for github repos.